static/templates/settings/organization_settings_admin.hbs

```diff
@@ -1,6 +1,22 @@
 <div id="organization-settings" data-name="organization-settings" class="settings-section">
     <form class="form-horizontal admin-realm-form org-settings-form">
 
+        {{#if (eq realm_plan_type 1)}}
+        <div id="org-pub-listing" class="org-subsection-parent">
+            <div class="subsection-header">
+                <h3>{{t "Public Listing" }}</h3>
+                {{> settings_save_discard_widget section_name="pub-listing" }}
+            </div>
+            <div class="inline-block organization-settings-parent">
+                {{> settings_checkbox
+                  setting_name="oss_showcase"
+                  prefix="id_"
+                  is_checked=oss_showcase
+                  label=admin_settings_label.realm_oss_showcase}}
+            </div>
+        </div>
+        {{/if}}
```

Comment:
Let's make this a dropdown to match the data model.  There's a chance we'll add additional values in the future that controls details like "How does the organization get listed?".